### ISSUE TITLE
Banned from UTRS 2/Unban

### ISSUE THREAD

**Issue by GrooveyDude** [UTRS2/utrs#600]
I have been blocked from editing multiple times for using original research on Wikipedia and not using citations. I only evaded a block once and later got banned from UTRS for too many unblock requests during the pandemic. The ban ID is # 138 and my Wikipedia username is Groovey Dude. I won't do these things again and won't evade a block again. Please unban me. When I make edits it won't be original research and it will be from sources.

**Comment by dqwiki** [comment]
We don't handle bans here as a development team. You need to email utrs-admins@googlegroups.com
